concourse/client/util.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019-2020 SAP SE or an SAP affiliate company. All rights reserved. This file is
+# licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
```

Comment:
nit: by convention (and according to, I think, PEP-8), we group imports:

```
import <package or module from std python>

import <package or module from 3rd-party>

import <local package or module>
```

sorting imports alphabetically (group-wise), leaving no blank lines within a group, and exactly one blank line between groups

Reply:
done